README.md

```diff
@@ -22,6 +22,7 @@ You'll need to have Terraform & the AWS CLI installed locally, and an OKTA accou
     * export OKTA_BASE_URL=okta.com
     * export OKTA_API_TOKEN=12345678979654645646556
 
+If needed, read here [how to create an Okta API Token](https://bit.ly/get-okta-api-token).
```

Comment:
Housekeeping.

While I was able to create an Okta account, I was not sure about which type of token I had to setup (there are multiple options). I initially created the wrong one and the Terraform Okta module showed me an error message with this link that I'm adding.

After following the instructions I was able to create the correct one.